docs/source/index.mdx

```diff
@@ -1,17 +1,52 @@
+
+- **Community Leaderboards** bring together the best models for a given task or domain and make them accessible to everyone.
+- **Model Cards** provide a comprehensive overview of a model's capabilities from the author's perspective.
+- **Libaries and Packages** give you the tools to evaluate your models on the Hub.
+
+## Community Leaderboards
+
+Community leaderboard show how a model performs on a given task or domain. For example, their are leaderboards for question-answering, reasoning, classification, vision, and audio. If you're tackling a new task, you can use a leaderboard to see how a model performs on it.
+
+Here are some examples of community leaderboards:
+
+| Leaderboard | Task | Description |
+| --- | --- | --- |
+| [MTEB](https://huggingface.co/spaces/mteb/leaderboard)| Embedding | MTEB leaderboard compares 100+ text and image embedding models across 1000+ languages. We refer to the publication of each selectable benchmark for details on metrics, languages, tasks, and task types. Anyone is welcome to add a model, add benchmarks, help us improve zero-shot annotations or propose other changes to the leaderboard. |
+| [GAIA](https://huggingface.co/spaces/gaia-benchmark/leaderboard)| Agentic | GAIA is a benchmark which aims at evaluating next-generation LLMs (LLMs with augmented capabilities due to added tooling, efficient prompting, access to search, etc). (See our paper for more details.) |
+| [OpenVLM Leaderboard](https://huggingface.co/spaces/opencompass/open_vlm_leaderboard)| Vision Language Models | The OpenVLM Leaderboard evaluates 272+ Vision-Language Models (including GPT-4v, Gemini, QwenVLPlus, LLaVA) across 31 different multi-modal benchmarks using the VLMEvalKit framework. It focuses on open-source VLMs and publicly available API models. |
+| [Open ASR Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard)| Audio | The Open ASR Leaderboard ranks and evaluates speech recognition models on the Hugging Face Hub. Models are ranked based on their Average WER, from lowest to highest. |
+| [LLM-Perf Leaderboard](https://huggingface.co/spaces/llm-perf/leaderboard)| LLM Performance | The 🤗 LLM-Perf Leaderboard 🏋️ is a leaderboard at the intersection of quality and performance. Its aim is to benchmark the performance (latency, throughput, memory & energy) of Large Language Models (LLMs) with different hardwares, backends and optimizations using Optimum-Benhcmark. |
+
+There are tonnes more leaderboards on the Hub. Check out all the leaderboards via this [search](https://huggingface.co/spaces?category=model-benchmarking) or use this [dedicated space](https://huggingface.co/spaces/OpenEvals/find-a-leaderboard) to find a leaderboard for your task.
+
+## Model Cards
+
+Model cards provide an overview of a model's capabilities evaluated by the model's author. They are a great way to understand a model's capabilities and limitations.
```

Comment:
Not always evaluated by the model's author/developer! There are community contributions too!